great_expectations/render/renderer/content_block/content_block.py

```diff
@@ -35,11 +35,15 @@ def render(cls, render_object, **kwargs):
 
         if isinstance(render_object, list):
             blocks = []
+            has_failed_evr = False if type(render_object[0]) == ExpectationValidationResult else None
             for obj_ in render_object:
                 expectation_type = cls._get_expectation_type(obj_)
 
                 content_block_fn = cls._get_content_block_fn(expectation_type)
 
+                if type(obj_) == ExpectationValidationResult and not obj_.success:
```

Comment:
```suggestion
                if isinstance(obj_, ExpectationValidationResult) and not obj_.success:
```

This allows us to be friendlier to subclassers

Reply:
Done!